heat/core/linalg.py

```diff
@@ -877,3 +877,87 @@ def triu(m, k=0):
         Upper triangle of the input tensor.
     """
     return __tri_op(m, k, torch.triu)
+
+def norm(w,norm_order=2):
+    """
+        Calculates the norm of a given vector
+    """
+    beta = abs(max(w[0], key=abs))
```

Comment:
These should probably be methods of heat, i.e. ht.abs()/DNDarray.abs() and ht.max()/DNDarray.max().